lpwan-backend/src/main/java/com/cumulocity/lpwan/util/ApplicationContextProviderUtil.java

```diff
@@ -7,7 +7,7 @@
 
 @Component
 public class ApplicationContextProviderUtil implements ApplicationContextAware {
-    private static ApplicationContext ctx;
+    public static ApplicationContext ctx;
```

Comment:
Why is the scope change needed? In tests we can use setApplicationContext

Reply:
Sorry, I pushed this commit by mistake. I have revereted it.